dist/src/main/java/io/camunda/zeebe/gateway/GatewayConfiguration.java

```diff
@@ -46,6 +47,15 @@ public MultiTenancyCfg multiTenancyCfg() {
     return config.getMultiTenancy();
   }
 
+  @Bean
+  public SchedulerConfiguration schedulerConfiguration() {
+    final var cpuThreads = config.getThreads().getManagementThreads();
+    final var ioThreads = 0;
```

Comment:
```suggestion
    // We set ioThreads to zero as the Gateway isn't using any IO threads.
    final var ioThreads = 0;
```

Reply:
Actually in general I think this is not really true, we could move some to ioThreads for requests etc as they io heavy, but right now we don't. AFAIK